cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,45 @@
-describe('', () => {
-  before(() => {
+describe('Article creation and deletion', () => {
+  let user;
+  let article;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
```

Comment:
I can't find declaration of ```generateUser```
How is it supposed to work?

Reply:
Still can't find this method/task. Make sure you have added into commit all the required imports